src/OctoshiftCLI.Tests/AdoApiTests.cs

```diff
@@ -238,6 +239,7 @@ public async Task GetGithubAppId_Should_Return_Null_When_No_Team_Projects_Have_E
             var githubOrg = "foo-gh-org";
             var teamProject1 = "foo-tp1";
             var teamProject2 = "foo-tp2";
+            var teamProjectArg = string.Empty;
```

Comment:
depending on what we do with the previous comment, we may want a test for the scenario where teamProjectArg is not empty.